src/Directory.Build.props

```diff
@@ -21,7 +21,7 @@
     
     <!-- Defaults for target frameworks and architecture -->
     <LibraryTargetFrameworks>netstandard2.0</LibraryTargetFrameworks>
-    <LibraryTargetFrameworks Condition="'$(OsEnvironment)'=='windows'">net46;netstandard2.0</LibraryTargetFrameworks>
+    <LibraryTargetFrameworks Condition="'$(OsEnvironment)'=='windows'  or '$(DotNetBuildFromSource)'=='true'">net46;netstandard2.0</LibraryTargetFrameworks>
```

Comment:
Why wouldn't we just remove the condition completely now?

Reply:
Removed the condition.